integration-tests/sql/src/main/java/org/apache/camel/quarkus/component/sql/it/SqlHelper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.sql.it;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SqlHelper {
+
+    private static Set<String> BOOLEAN_AS_NUMBER = new HashSet<>() {
+        {
+            add("db2");
+            add("mssql");
+            add("oracle");
+
+        }
+    };
```

Comment:
This is OK in a test, but it's good to know that double bracket initialization may have a performance impact due to introducing a new type https://www.youtube.com/watch?v=wgJWs14YcEs&t=1504s More about [ mono | bi | poly | mega ]morphic call sites https://www.youtube.com/watch?v=wgJWs14YcEs&t=483s
`new HashSet<>(Arrays.asList("db2", "mssql", "oracle"))` might be generally a better choice

Reply:
Thanks for the information, I  didn't know that.

Reply:
done